src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,49 @@
+package racingcar.model;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+public class Car {
+	private static final int MAX_NAME_LENGTH = 5;
+    private static final int MIN_NAME_LENGTH = 1;
+    private static final String NAME_FORMAT_REGEX = "^[a-zA-Z0-9]*$";
+    private final String name;
+    private int position;
+
+    public Car(String name, int position) {
+        validateNameLength(name);
+        validateEachCharacter(name);
+        this.name = name;
+        this.position = position;
+    }
+
+    private void validateNameLength(String name) {
+        if (name.length() > MAX_NAME_LENGTH || name.length() < MIN_NAME_LENGTH) {
+            throw new IllegalArgumentException(
+                    String.format("자동차 이름은 %d~%d 사이여야 합니다.", MIN_NAME_LENGTH, MAX_NAME_LENGTH)
+            );
+        }
+    }
```

Comment:
```suggestion
    private void validateNameLength(String name) {
        if (name.length() < MIN_NAME_LENGTH || name.length() > MAX_NAME_LENGTH) {
            throw new IllegalArgumentException(
                    String.format("자동차 이름은 %d~%d 사이여야 합니다.", MIN_NAME_LENGTH, MAX_NAME_LENGTH)
            );
        }
    }
```

사소하지만 최소값 검증이 앞에 있으면 더 읽기 쉬운 코드가 될 것 같습니다!